system/libuv/Kconfig

```diff
@@ -6,6 +6,7 @@
 config LIBUV
 	bool "libuv asynchronous I/O Library"
 	default n
+	select PIPES
```

Comment:
```suggestion
	depends on PIPES
```

Reply:
why do you prefer depends-on?
i suspect select is a bit more user-friendly for things like this.

Reply:
done.
i'm still interested in why depends-on is preferred.

Reply:
> please reference: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-recommendations

thank you.
it was not convincing me though.

Reply:
good question :-) maybe we should discuss the coherent policy on the mailing list for NuttX too? :-)

Reply:
Thank you @mlyszczek for perfect explanation with examples!! :-)